Resources/Prototypes/Body/Parts/animal.yml

```
@@ -58,6 +59,7 @@
     - state: r_leg
   - type: BodyPart
     partType: Leg
+    slotId: legs # Shitmed
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)